docs/dev/reference/dca/list.md

```diff
@@ -22,7 +22,7 @@ $GLOBALS['TL_DCA']['tl_example']['list']['sorting'] = [
 
 | Key                   | Value                            | Description                                                                                                                                                                                                                                                                                                                                                                                            |
 |-----------------------|----------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| mode                  | Sorting mode (`integer`)         | **0** Records are not sorted <br>**1** Records are sorted by a fixed field <br>**2** Records are sorted by a switchable field <br>**3** Records are sorted by the parent table <br>**4** Displays the child records of a parent record (see style sheets module) <br>**5** Records are displayed as tree (see site structure) <br>**6** Displays the child records within a tree structure (see articles module)               |
+| mode                  | Sorting mode (`integer`)         | **0** Records are not sorted <br>**1** Records are sorted by a fixed field <br>**2** Records are sorted by a switchable field <br>**3** Records are sorted by the parent table <br>**4** Displays the child records of a parent record (see style sheets module) <br>**5** Records are displayed as tree (see site structure) <br>**6** Displays the child records within a tree structure (see articles module)<br><br>__Note:__ When using mode **4**, you will need to implement [child_record_callback](../callbacks#list-sorting-child-record), otherwise no records will be shown in backend.               |
```

Comment:
```suggestion
| mode                  | Sorting mode (`integer`)         | **0** Records are not sorted <br>**1** Records are sorted by a fixed field <br>**2** Records are sorted by a switchable field <br>**3** Records are sorted by the parent table <br>**4** Displays the child records of a parent record (see style sheets module) <br>**5** Records are displayed as tree (see site structure) <br>**6** Displays the child records within a tree structure (see articles module)<br><br>__Note:__ When using mode **4**, you will need to implement a [child_record_callback](../callbacks#list-sorting-child-record), otherwise no records will be shown in backend.               |
```